engine/apps/schedules/tasks/__init__.py

```diff
@@ -1,18 +1,19 @@
-from .drop_cached_ical import drop_cached_ical_for_custom_events_for_organization, drop_cached_ical_task  # noqa: F401
-from .notify_about_empty_shifts_in_schedule import (  # noqa: F401
```

Comment:
I'm _fairly_ certain this leads to an issue for [these imports](https://github.com/grafana/oncall/blob/dev/engine/apps/schedules/tasks/refresh_ical_files.py#L5):
```python3
from apps.schedules.tasks import notify_about_empty_shifts_in_schedule, notify_about_gaps_in_schedule
```

It appears there is a naming clash between _modules_ and _functions_ with the same name here. This ☝️ `import` statement is importing the _modules_ `notify_about_empty_shifts_in_schedule` and `notify_about_gaps_in_schedule` from the `engine/apps/schedules/tasks` directory _not_ the similarly named functions that're imported into `engine/apps/schedules/tasks/__init__.py`.